test/integration/targets/acme_certificate/tasks/main.yml

```diff
@@ -0,0 +1,244 @@
+---
+- name: store the ACME test container IP
+  set_fact:
+    acme_host: "{{ lookup('env', 'acme_host') }}"
+- name: register openssl version
+  shell: "openssl version | cut -d' ' -f2"
+  register: openssl_version
+- debug: msg="ACME test container IP is {{ acme_host }}; OpenSSL version is {{ openssl_version.stdout }}"
```

Comment:
For common tasks that aren't included in the test plugin, you can consolidate them in a `setup_acme` target and make it a dependency of the test targets. Example:

https://github.com/ansible/ansible/blob/3c72161da5a9712726e755ec02fe2ca86abf9d86/test/integration/targets/ec2_key/meta/main.yml#L4

You can also use aliases to define dependencies:

* `setup/once/{target_name}` - `target_name` runs once for the entire test run.
* `setup/always/{target_name}` - `target_name` runs on each test target.